packaging/language/maven_artifact.py

```diff
@@ -240,12 +246,14 @@ def _request(self, url, failmsg, f):
                 client = boto3.client('s3',aws_access_key_id=self.module.params.get('username', ''), aws_secret_access_key=self.module.params.get('password', ''))
                 url_to_use = client.generate_presigned_url('get_object',Params={'Bucket':bucket_name,'Key':key_name},ExpiresIn=10)
 
+        req_timeout = self.module.params.get('timeout', '10')
```

Comment:
the alternative value `'10'` should either be an int or even be removed as there is already a default in the arg spec.